src/Roslyn.Suppressions/Roslyn.Suppressions/Roslyn.Suppressions.csproj

```diff
@@ -24,6 +24,17 @@
   
   <ItemGroup>
     <ProjectReference Include="..\..\Core\Core.csproj" />
+    <ProjectReference Include="..\..\Infrastructure.VS\Infrastructure.VS.csproj" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
```

Comment:
For some reason this was added automatically by VS, perhaps it has to be on the csproj directly and not in the targets file?

Reply:
It may just be VS being stupid. If we remove it, VS will just keep adding it back.
Probably the best we can do is make it clearer why they are there e.g.

```
  <ItemGroup Label="Automatically added by VS">
    <!-- Note: these settings relate to files in the SuppressorCodeGen.targets -->
    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
    <Compile Update="SupportedSuppressionBuilder.g.cs">
      <DesignTime>True</DesignTime>
    </Compile>
  </ItemGroup>
```